### DISCUSSION TITLE
Guideline and line position

### DISCUSSION THREAD

**Discussion by neobie** [patrykandpatrick/vico#774] [Questions]
### Question

<img width="355" alt="image" src="https://github.com/patrykandpatrick/vico/assets/4213973/19a6ebbc-f795-4bd0-ac58-4ca9bdb88153">

How to make the guideline to appear 3 lines only instead of 5 ? I can't find guideline related "count" property.
And make the line to start from the left ?

### Vico version(s)

1.15

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
>How to make the guideline to appear 3 lines only instead of 5 ?

This is controlled by `AxisItemPlacer.Horizontal`, but we need more context to give you a more specific answer.
Are guidelines supposed to be displayed for some particular _x_ values?

>And make the line to start from the left ?

Apply `HorizontalLayout.FullWidth`.
